app/views/browse/deployment-config.html

```diff
@@ -138,6 +138,10 @@
                             <span ng-if="!('deploymentconfigs' | canI : 'update')">none</span>
                           </p>
                           <volumes volumes="deploymentConfig.spec.template.spec.volumes" namespace="project.metadata.name"></volumes>
+                          <p ng-if="deploymentConfig.spec.template.spec.volumes.length">
+                            <a ng-if="'deploymentconfigs' | canI : 'update'"
```

Comment:
add the canI check to the ng-if above so you don't end up with an empty `<p>` when you dont have update rights

```
ng-if="deploymentConfig.spec.template.spec.volumes.length && 'deploymentconfigs' | canI : 'update'"
```